types/titanium/tsconfig.json

```diff
@@ -2,8 +2,7 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "dom",
-            "es6"
+            "es5"
```

Comment:
I don't usually see this go backwards.  Intentional?

Reply:
Yes, these were wrong before. `dom` was only used to get typings for `console`, which are now included in our typings. Also the typings do not use any ES6 features yet, so we changed that as well and only include `es5`.